adapter/internal/discovery/xds/server.go

```diff
@@ -229,25 +231,23 @@ func GetEnforcerThrottleDataCache() wso2_cache.SnapshotCache {
 func DeleteAPICREvent(vHosts, labels []string, apiUUID string, organizationID string) error {
 	mutexForInternalMapUpdate.Lock()
 	defer mutexForInternalMapUpdate.Unlock()
-	for _, vhost := range vHosts {
-		apiIdentifier := GenerateIdentifierForAPIWithUUID(vhost, apiUUID)
-		if err := deleteAPI(apiIdentifier, labels, organizationID); err != nil {
-			logger.LoggerXds.ErrorC(logging.ErrorDetails{
-				Message: fmt.Sprintf("Error undeploying API %v of Organization %v from environments %v",
-					apiIdentifier, organizationID, labels),
-				Severity:  logging.MAJOR,
-				ErrorCode: 1410,
-			})
-			return err
-		}
-		// if no error, update internal vhost maps
-		// error only happens when API not found in deleteAPI func
-		logger.LoggerXds.Infof("Successfully undeployed the API %v under Organization %s and environment %s ",
-			apiIdentifier, organizationID, labels)
-		for _, environment := range labels {
-			// delete environment if exists
-			delete(apiUUIDToGatewayToVhosts[apiUUID], environment)
-		}
+	apiIdentifier := apiUUID
```

Comment:
why require new var? and this is misleading the apiIdentifier name as we have used it in other places in code for combination of vhost + apiid